panels/updates/cc-updates-panel.c

```diff
@@ -156,6 +157,12 @@ store_automatic_updates_setting (NMConnection *connection,
   setting_user = ensure_setting_user (connection);
   g_assert (setting_user != NULL);
 
+  /* Calling nm_setting_user_set_data() causes notifications from NM. */
+  if (self->current_device && self->changed_id)
+    g_signal_handler_block (self->current_device, self->changed_id);
+  if (self->current_connection && self->connection_changed_id)
+    g_signal_handler_block (self->current_connection, self->connection_changed_id);
+
```

Comment:
Do we ever call this function before connecting to these signals? It wouldn't be how I imagined its usage to be, and I'd rather have a `g_assert(self->current_device && self->changed_id > 0)` than these `if`s.

Reply:
I guess that's fine for now.